policy/approval/approve.go

```diff
@@ -296,6 +304,28 @@ func (r *Rule) filterEditedCandidates(ctx context.Context, prctx pull.Context, c
 	return allowedCandidates, nil
 }
 
+func (r *Rule) filterEditedBodyCandidates(ctx context.Context, prctx pull.Context, candidates []*common.Candidate) ([]*common.Candidate, error) {
```

Comment:
With the exception of the flag that's checked and part of a log message, this function seems identical to `filterEditedCommentCandidates`. I think it would be good to create a single function that can handle both case. Because both of these functions are only called after checking the option value, I don't think there's a reason to additionally check the option again within the function, so that removes one difference. For the log message, maybe add a `candidateType` parameter or similar?

This also raises a question about the `ignore_edited_body` option. Do you think it is helpful to control body edits independently from comment edits or would a single option (the existing `ignore_edited_comments`) be sufficient?

Reply:
> This also raises a question about the ignore_edited_body option. Do you think it is helpful to control body edits independently from comment edits or would a single option (the existing ignore_edited_comments) be sufficient?

that's an excellent question. I *thiink* with how we wanted to use PR body values it would be ideal to have them function as separate entities. But I can see how it would also make sense to lump the pr into the single option of `ignore_edited_comments`.

I'll refactor the method to use a single one though and see how it looks from there.